layer/frame_time_layer.cc

```diff
@@ -117,6 +146,23 @@ void CreateFinishIndicatorFile(const char* finishCause) {
   fclose(finish_file);
 }
 
+bool FrameTimeLayerData::HasBenchmarkStarted() {
+  if (benchmark_state_idx_ > 0 || benchmark_start_pattern_.empty()) return true;
+
+  if (!benchmark_log_scanner_.has_value() || !benchmark_log_scanner_->is_open())
+    return true;
+
+  if (benchmark_log_scanner_->consume_new_lines()) {
+    benchmark_state_idx_ = 1;
```

Comment:
a bool seems more appropriate if benchmark_state_idx_ can only be 0 or 1 (or was this supposed to receive the value of get_first_occurrence_line_num()?)

Reply:
I intent to extend this to detecting multiple game states, e.g, initialization, in-menu, in-level, etc. While a `bool` makes sense now, I didn't want to have to refactor this in near future.